helpers/exceptions.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 module.exports = {
+	mainnetGenesisPublicKey: 'd121d3abf5425fdc0f161d9ddb32f89b7750b4bdb0bff7d18b191d4b4bafa6d4',
```

Comment:
as we don't have genesis account secret accessible I don't think that sending transaction by it is testable.